RecCalorimeter/src/components/ClusterJet.h

```diff
@@ -0,0 +1,79 @@
+#ifndef RECCALORIMETER_CLUSTERJET_H
+#define RECCALORIMETER_CLUSTERJET_H
+
+// std
+#include <map>
+
+/*
+// Gaudi
+#include "GaudiKernel/ToolHandle.h"
+#include "GaudiKernel/MsgStream.h"
+#include "GaudiKernel/StatusCode.h"
+#include "GaudiAlg/Transformer.h"
+#include "k4FWCore/BaseClass.h"
+
+// k4FWCore
+#include "k4FWCore/DataHandle.h"
+
+#include "edm4hep/ReconstructedParticleCollection.h"
+#include "edm4hep/MCParticleCollection.h"
+*/
+
+#include "fastjet/JetDefinition.hh"
+
+
+// Gaudi
+#include "GaudiAlg/GaudiAlgorithm.h"
+
+
+// EDM4hep
+#include "edm4hep/ClusterCollection.h"
+
+
+
+
+class ClusterInfo : public fastjet::PseudoJet::UserInfoBase{
+ public:
+ ClusterInfo(const int & index) : _index(index){}
+  int index() const{ return _index; }
+ protected:
+  int _index;
+};
+
+
+
+/** @class ClusterJet k4RecCalorimeter/RecCalorimeter/src/components/ClusterJet.h
+ *
```

Comment:
Can you describe the tool here? Adding what an end-user should know to properly configure/use it without having to read the code?

Reply:
Done